src/main/java/com/learn/java/inheritance/AbstractFlyer.java

```diff
@@ -0,0 +1,7 @@
+package com.learn.java.inheritance;
+
+public abstract class AbstractFlyer implements Flyer {
+    public void fly(){
+        System.out.println(1);
```

Comment:
modify the fly method in order to receive an argument _flyer_ and print "the _flyer_ flies"